tratto/src/main/java/star/tratto/util/JavaTypes.java

```diff
@@ -98,7 +98,7 @@ public class JavaTypes {
     );
     public static final List<String> TYPICAL_TYPES = TYPICAL.stream().map(Pair::getValue1).collect(Collectors.toList());
```

Comment:
What does "typical" mean here and above?  Can you please replace it by a more descriptive word?